ubuntu/standard/Dockerfile.template

```diff
@@ -67,12 +71,24 @@ RUN apt-get update \
  && apt-get install -y --no-install-recommends \
     maven \
  && rm -rf /var/lib/apt/lists/*
+ENV ANT_HOME=/usr/share/ant \
+    GRADLE_HOME=/usr/share/gradle \
+    M2_HOME=/usr/share/maven \
+    JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8
 
 # Install Go
-RUN curl -sL https://dl.google.com/go/go1.9.2.linux-amd64.tar.gz -o go1.9.2.linux-amd64.tar.gz \
- && mkdir -p /usr/local/go1.9.2 \
- && tar -C /usr/local/go1.9.2 -xzf go1.9.2.linux-amd64.tar.gz --strip-components=1 go \
- && rm go1.9.2.linux-amd64.tar.gz
+RUN curl -sL https://dl.google.com/go/go1.8.6.linux-amd64.tar.gz -o go1.8.6.linux-amd64.tar.gz \
+ && mkdir -p /usr/local/go1.8.6 \
+ && tar -C /usr/local/go1.8.6 -xzf go1.8.6.linux-amd64.tar.gz --strip-components=1 go \
+ && rm go1.8.6.linux-amd64.tar.gz
+RUN curl -sL https://dl.google.com/go/go1.9.3.linux-amd64.tar.gz -o go1.9.3.linux-amd64.tar.gz \
+ && mkdir -p /usr/local/go1.9.3 \
+ && tar -C /usr/local/go1.9.3 -xzf go1.9.3.linux-amd64.tar.gz --strip-components=1 go \
+ && rm go1.9.3.linux-amd64.tar.gz
+ENV GOROOT_1_8_X64=/usr/local/go1.8.6 \
```

Comment:
Is there a standard environment variable naming for setting the GOROOT for multiple versions? Or is this just something you created so that people can reference it from their tasks/scripts?

Reply:
These variables are non-standard.  They're for referencing from tasks/scripts like you say.  The similar Java variables let the Maven and Gradle tasks find a specific version of Java when users select that version in their build definition.

Reply:
Ok, that's what I thought - thanks.